### DISCUSSION TITLE
Dashboard.yaml file posted here

### DISCUSSION THREAD

**Discussion by STGMavrick** [dinki/View-Assist#74] [Q&A]
Hi - I've been following the setup guide along with the videos and they've been amazing! I'm working on the dashboard section where we copy/paste the contents of dashboard.yaml posted here. 

The issue I'm having is with the files indentation. Directly copy/paste into the new "title Alexa" section from the guide doesn't pass the edit checker. The indentation needs to be further out. Is there an easy way to increase the indentation of all 700 lines before pasting it into my dashboard yaml?

**Comment by dinki** [comment]
Hi there.  Here to help but I'm not sure I understand.  You should be able to just copy and paste the raw code into the dashboard while in the raw configuration editor.  Not sure what 'title alexa' means.

I'll encourage you to join us on Discord where it is much easier/faster to diagnosis and suggest things for problems like these.  We are a friendly bunch for sure.
